lib/roadie/style_block.rb

```diff
@@ -9,24 +9,33 @@ class StyleBlock
 
     # @param [Selector] selector
     # @param [Array<StyleProperty>] properties
-    def initialize(selector, properties)
+    def initialize(selector, properties, media)
       @selector = selector
       @properties = properties
+      @media = media
     end
 
     # @!method specificity
     #   @see Selector#specificity
-    # @!method inlinable?
-    #   @see Selector#inlinable?
-    def_delegators :selector, :specificity, :inlinable?
+    def_delegators :selector, :specificity
     # @!method selector_string
     #   @see Selector#to_s
     def_delegator :selector, :to_s, :selector_string
 
     # String representation of the style block. This is valid CSS and can be
     # used in the DOM.
+    def inlinable?
+      inlinable_media? && selector.inlinable?
+    end
+
     def to_s
       "#{selector}{#{properties.map(&:to_s).join(';')}}"
     end
+
+    private
+
+    def inlinable_media?
+      @media.all? {|media_query| media_query.to_s.count('()') < 2 }
```

Comment:
Please add a space to the left of the `|block parameter|` pipe.

(I separate blocks from hashes using spaces: `{ block }`, `{hash: 1}`.)

---

How come you check for both opening and closing parens? Shouldn't this be enough?

```ruby
@media.none? { |media| media.to_s.include? '(' }
```

This should also be faster since it's less work to do.

Did I miss something? :-)